api/database/migrations/2022_12_01_215438_add_self_declaration_fields.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+use Database\Helpers\ApiEnums;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('indigenous_declaration_signature')->nullable()->default(null);
+            $table->jsonb('indigenous_communities')->nullable()->default(null);
+        });
+
+        DB::statement(
+            <<<SQL
+                UPDATE users
+                    SET indigenous_communities =
```

Comment:
I suggest using a similar case statement to fill the signaure field too.  We'll probably have some kind of "is non-null" logic on this field eventually so I suggest filling it with a string like "migrated 2022-12-01" or something.

Reply:
sure https://github.com/GCTC-NTGC/gc-digital-talent/pull/4994/commits/ac7ba6a357ba3e59b4216a86f8f3d22107021fab